communication/youtube/youtube-guidelines.md

```diff
@@ -174,6 +174,11 @@ Example description:
 Kubernetes Contributor Experience Special Interest Group Weekly Meeting. Check
 here (link to sig list) for a complete list of SIGs and when they meet. Join us!
 
+Please note the following items:
+* The date must be in title and description
+* The date format must be YYYYMMDD
+* The section titled `Is this video made for kids?` should not be altered for search reasons
+
```

Comment:
Could you add a couple examples of Good vs Bad titles?

Reply:
Yup, let me do that now